src/gui/src/displayControls.cpp

```diff
@@ -381,6 +391,36 @@ DisplayControls::~DisplayControls()
   custom_controls_.clear();
 }
 
+void DisplayControls::createLayerMenu()
+{
+  connect(layers_menu_->addAction("Only show selected"),
+          &QAction::triggered,
+          [this]() {
+            layerShowOnlySelectedNeighbors(0, 0);
+          });
+  auto* neighbors = layers_menu_->addMenu("Show with neighboring layers");
+  connect(neighbors->addAction("1 above / 1 below"),
```

Comment:
Perhaps "Show layer range +/- 1" or is that too cryptic?  The others would be similar.

Reply:
+-1 might be okay, another possibility would be: Show layer range 1 above and below (this might make the menu pretty wide though)

Reply:
Another option is to use unicode up/down arrows, something like

Show layer range ^v
Show layer range ^^vv  (or Show layer range ^2v2)
Show layer range ^
Show layer range v

or superscript/subscript

Reply:
I think "Show layer range 1 above and below" ok too

Reply:
I think the arrows does a good job of balancing the size of the menu with the information. I made the arrows follow how the layers are layed out in the display controls (ie. ^ means below) since its pointing in the direction of the layer (but I can swap it if needed or you think that is confusing.

Reply:
I like it.  Maybe we could use the UP DOWN ARROW (U+2195) ↕ rather than ^v

I think the arrows will become clear once someone uses them once.

Reply:
I made the ^v an updown arrow.